### PR TITLE
Add procedure for selecting the JDK when installed via RPMs

### PR DESCRIPTION
**Pull request by jerboaa**

### CODE REVIEW

File: modules/proc-rhel-selecting-jdk-rpm.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
+[id="rhel_select_jdk_rpm"]
+= Selecting a specific JDK on {rhel} which were installed using the package manager `yum`
```
Comment (robin-owen) [inline modules/proc-rhel-selecting-jdk-rpm.adoc]: This is a very long title -- we could probably remove "which were installed using the package manager `yum`", as this is detailed in the prerequisites.

File: modules/proc-rhel-selecting-jdk-rpm.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+If you have multiple JDKs installed on {rhel} you can select a
+specific JDK using the `alternatives` command. Selecting the JDK by using this
+approach changes the default `java` system wide and requires root privileges.
+See xref:proc-rhel-setting-java-home-env-variable.adoc[setting `JAVA_HOME` variable]
+documentation if you need to select a JDK without root privileges or for
+per-application selection of a specific JDK.
+
+.Prerequisites
+. xref:proc-rhel-installing-multiple-major-versions.adoc[Have multiple JDKs installed] on your system using the `yum` package manager.
```
Comment (robin-owen) [inline modules/proc-rhel-selecting-jdk-rpm.adoc]: Technically, a user with `root` permissions would also be a prerequisite, correct?

File: modules/proc-rhel-selecting-jdk-rpm.adoc (hunk 1 above)
Comment (robin-owen) [reply modules/proc-rhel-selecting-jdk-rpm.adoc]:
```suggestion
= Selecting a specific JDK on {rhel}
```

File: modules/proc-rhel-selecting-jdk-rpm.adoc (hunk 2 above)
Comment (jerboaa) [reply modules/proc-rhel-selecting-jdk-rpm.adoc]: yes. will fix.

File: modules/proc-rhel-selecting-jdk-rpm.adoc (hunk 1 above)
Comment (jerboaa) [reply modules/proc-rhel-selecting-jdk-rpm.adoc]: Makes sense, I'll fix.

**Comment by jerboaa** [comment]
@robin-owen How about now?

**Comment by ncbaratta** [comment]
Looks great!

File: modules/proc-rhel-selecting-jdk-rpm.adoc (hunk 1 above)
Comment (jerboaa) [reply modules/proc-rhel-selecting-jdk-rpm.adoc]: Note that there is multiple ways to select a JDK on RHEL. So `Selecting a specific JDK on {rhel}` is not specific enough. It needs to mention "installed via RPMs" or "installed via yum" or some such.

**Pull request merged**